packages/v4/src/content/design-guidelines/usage-and-behavior/usage-and-behavior.md

```diff
@@ -0,0 +1,119 @@
+---
+id: Usage and behavior
+section: guidelines
+---
+
+## Displaying structured data
+Structured data includes any information that is stored in a database or similar regular data structure. The most common ways of presenting structured data are to display it in either a list or a table where rows correspond to records in the database. PatternFly supports two controls for displaying this type of data: the [Data list](/components/data-list) and the [Table](/components/table).
```

Comment:
You capitalize data list and table here but in the rest of the page when you refer to components, they are sentence cased! Think these should be sentence cased as well for consistency